src/Vlingo.Xoom/SystemHolder.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright © 2012-2020 VLINGO LABS. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Vlingo.Xoom
+{
+    public class SystemHolder
```

Comment:
What this is supposed to do? Shouldn't it load properties from the file ? Shouldn't it be using `Settings.cs` or something in `LoadVariables` method ?

Reply:
It was created to keep environment variables. It loads the variables at run time by LoadVariables method.